servers/src/main/java/tachyon/master/file/meta/MountTable.java

```diff
@@ -138,7 +138,7 @@ public synchronized String getMountPoint(TachyonURI uri) throws InvalidPathExcep
    * @param uri a Tachyon path URI
    * @return mount point the given Tachyon path is nested under
    */
-  public synchronized boolean isMountPoint(TachyonURI uri) throws InvalidPathException {
+  public synchronized boolean isMountPoint(TachyonURI uri) {
```

Comment:
It removes unnecessary thrown error, which will make its caller (this PR added several callers) error handling logic easier.